examples/mlserver/client.py

```diff
@@ -0,0 +1,27 @@
+import requests, threading
+
+NUM_THREADS = 2
+URL = "http://localhost:8080/v2/models/text-classification-model/infer"
+sentences = ["I hate using GPUs for inference", "I love using DeepSparse on CPUs"] * 100
```

Comment:
Should `* 100` be `* NUM_THREADS` if we are taking only `sentences[:NUM_THREADS]` elements?

Reply:
@rsnm2 see suggestion below